packages/mux-player/src/media-theme-mux.html

```diff
@@ -485,157 +488,269 @@
-      <media-control-bar>
-        {{>PlayButton}}
-        <slot name="seek-live"></slot>
-        {{>SeekBackwardButton}} {{>SeekForwardButton}} {{>TimeDisplay}} {{>TimeRange}} {{>MuteButton}} {{>VolumeRange}}
-        {{>PlaybackRateButton}} {{>AirplayButton}} {{>CastButton}}
-      </media-control-bar>
-    </template>
+      <template if="streamType == 'live'">
+
+        <template if="targetLiveWindow > 0">
+          <template if="title">
+            <media-control-bar>{{>TitleDisplay}}</media-control-bar>
+          </template>
+          <media-control-bar>
+            {{>PlayButton}}
+            <slot name="seek-live"></slot>
+            {{>SeekBackwardButton}}
+            {{>SeekForwardButton}}
+            {{>TimeDisplay}}
+            {{>TimeRange}}
+            {{>MuteButton}}
+            {{>VolumeRange}}
+            {{>PlaybackRateButton}}
+            {{>AirplayButton}}
+            {{>CastButton}}
+          </media-control-bar>
+        </template>
+
+        <template if="targetLiveWindow == null">
+          <template if="title">
+            <media-control-bar>{{>TitleDisplay}}</media-control-bar>
+          </template>
+          <media-control-bar>
+            {{>PlayButton}}
+            <slot name="seek-live"></slot>
+            {{>MuteButton}}
+            {{>VolumeRange}}
+            <div class="spacer"></div>
+            {{>AirplayButton}}
+            {{>CastButton}}
+          </media-control-bar>
+        </template>
 
-    <template if="layout == 'audio live'">
-      <template if="title">
-        <media-control-bar>{{>TitleDisplay}}</media-control-bar>
       </template>
-      <media-control-bar>
-        {{>PlayButton}}
-        <slot name="seek-live"></slot>
-        {{>MuteButton}} {{>VolumeRange}}
-        <div class="spacer"></div>
-        {{>AirplayButton}} {{>CastButton}}
-      </media-control-bar>
     </template>
 
-    <template if="layout == 'on-demand extra-small'">
-      {{>TimeRange}}
-      <media-control-bar>
-        {{>PlayButton}} {{>MuteButton}}
-        <div class="spacer"></div>
-        {{>CaptionsButton}} {{>FullscreenButton}}
-      </media-control-bar>
-    </template>
-
-    <template if="layout == 'on-demand small'">
-      <media-control-bar slot="top-chrome"> {{>TitleDisplay}} </media-control-bar>
-      <div slot="centered-chrome" class="center-controls">
-        {{>SeekBackwardButton section="center"}} {{>PlayButton section="center"}} {{>SeekForwardButton
-        section="center"}}
-      </div>
-      {{>TimeRange}}
-      <media-control-bar>
-        {{>PlayButton}} {{>TimeDisplay}} {{>MuteButton}} {{>VolumeRange}}
-        <div class="spacer"></div>
-        {{>PlaybackRateButton}} {{>CaptionsButton}} {{>AirplayButton}} {{>CastButton}} {{>PipButton}}
-        {{>FullscreenButton}}
-      </media-control-bar>
-    </template>
+    <template if="audio == null">
+
+      <template if="streamType == 'on-demand'">
+
+        <template if="breakpointSm == null">
```

Comment:
suggestion(non-blocking): Add html comments here indicating the player sizes. e.g.
```html
<!-- Extra Small UI -->
```

Reply:
thanks for the review!

I was thinking similar, only downside this would be added in the player bundle so opted to not add these.